tests/functional/adapter/materialization/test_view_security.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+from tests.functional.adapter.materialization.fixtures import (
+    seed_csv,
+    model_sql,
+    profile_yml,
+)
+
+
+class TestViewSecurity:
+    """
+    Methods in this class will be of two types:
+    1. Fixtures defining the dbt "project" for this test case.
+       These are scoped to the class, and reused for all tests in the class.
+    2. Actual tests, whose names begin with 'test_'.
+       These define sequences of dbt commands and 'assert' statements.
+    """
+
+    # configuration in dbt_project.yml
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "name": "example",
+            "models": {
+                "+materialized": "view",
+                "+view_security": "invoker"
+            }
+        }
+
+    # everything that goes in the "seeds" directory
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {
+            "seed.csv": seed_csv,
+        }
+
+    # everything that goes in the "models" directory
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "materialization.sql": model_sql,
+            "materialization.yml": profile_yml,
+        }
+
+    # The actual sequence of dbt commands and assertions
```

Comment:
Do we need both  tests in this class if they do the same thing?

- test_run_see_test
- test_buid

Reply:
Not really, I can delete `test_build`